Assets/Tests/InputSystem/CoreTests_Actions.cs

```diff
@@ -8057,6 +8057,39 @@ public void Actions_Vector2Composite_RespectsButtonPressurePoint()
         Assert.That(value.Value, Is.EqualTo(Vector2.left));
     }
 
+    [Test]
+    [Category("Actions")]
+    public void Actions_Vector2Composite_WithKeyboardKeys_CancelOnRelease()
```

Comment:
Excellent to see you've added tests to cover this functionality which seem to have been missing.